<launch>
    <node name="ariac_sim" pkg="nist_gear" type="gear.py" args="
          --load-moveit
          --development-mode
          -f $(find test_competitor)/config/tasks_config/kitting.yaml
          $(find test_competitor)/config/sensors_config/kitting_sensors.yaml" required="true" output="screen" />
</launch>